VirtoCommerce.PricingModule.Web/Controllers/Api/PricingModuleController.cs

```diff
@@ -365,27 +365,6 @@ public IHttpActionResult DeleteAssignments([FromUri] string[] ids)
             return StatusCode(HttpStatusCode.NoContent);
         }
 
-        /// <summary>
-        /// Delete pricelist assignments
-        /// </summary>
-        /// <remarks>Delete pricelist assignments by given criteria.</remarks>
-        /// <param name="criteria">Filter criteria</param>
-        /// <todo>Return no any reason if can't update</todo>
-        [HttpDelete]
-        [ResponseType(typeof(void))]
-        [Route("api/pricing/filteredAssignments")]
```

Comment:
We don't need to delete this method and move logic to the client side, we need to use this new API method, but without service extension - use search here, then paged deletion.

Reply:
Moved batch remove operation to controller